src/api/app/components/add_review_dropdown_component.rb

```diff
@@ -0,0 +1,32 @@
+class AddReviewDropdownComponent < ApplicationComponent
+  def initialize(bs_request:, user:)
+    super
+
+    @bs_request = bs_request
+    @user = user
+  end
+
+  def render?
```

Comment:
Using `can_add_reviews?` would clarify things IMHO

Reply:
Using `render?` is a built-in feature of view components. It doesn't have to be called in the view. See the documentation for [conditional rendering](https://viewcomponent.org/guide/conditional_rendering.html).